tests/conftest.py

```diff
@@ -102,13 +104,13 @@ def limited_access_auth_header(
     )
 
 
-@pytest.fixture(scope="session")
-def username() -> str:
+@pytest.fixture(scope="session", name="username")
+def username_generator() -> str:
```

Comment:
The same here, now, it looks like can be used as a standalone function but it is a fixture 🤷